cmd/server-mux.go

```diff
@@ -1,531 +0,0 @@
-/*
```

Comment:
Can you do server-mux refactoring in a separate PR ? (if adding pcks#12 support doesn't require that)